prediction_market_agent/agents/microchain_agent/functions_from_tools.py

```diff
@@ -0,0 +1,13 @@
+from langchain.tools import BaseTool
+from microchain import Function
+
+
+def microchain_function_from_tool(tool: BaseTool, example_args: list[str]) -> Function:
```

Comment:
`example_args` isn't required for langchain/openai tools, but is for microchain Functions